pkg/collector/corechecks/snmp/internal/report/report_device_metadata_test.go

```diff
@@ -317,11 +317,17 @@ func Test_metricSender_reportNetworkDeviceMetadata_withInterfaces(t *testing.T)
 		},
 	}
 
+	diagnosis := []metadata.DiagnosisMetadata{{ResourceType: "ndm_device", ResourceID: "1234", Diagnoses: []metadata.Diagnosis{{
```

Comment:
Shall rename the test to `Test_metricSender_reportNetworkDeviceMetadata_withDeivcesInterfacesDiagnoses` ?

Reply:
Done 👍